### PR TITLE
Fix/update recheck

### PR DESCRIPTION
**Pull request by diba1013**

### CODE REVIEW

File: src/test/java/de/retest/recheck/cli/testutils/TestReportCreator.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public static String createTestReportFileWithWarnings( final TemporaryFolder fol
 		final SuiteReplayResult suite = createSuiteReplayResultWithDiffs( folder );
 
 		final ElementIdentificationWarning elementIdentificationWarning =
-				new ElementIdentificationWarning( "someTestClass", null );
+				new ElementIdentificationWarning( "FooTest", null, null, "someTestClass" );
```
Comment (beatngu13) [inline src/test/java/de/retest/recheck/cli/testutils/TestReportCreator.java]: Don't know if you need this data at all, but if you pass something, I would try to get close to what is expected:

```suggestion
				new ElementIdentificationWarning( "FooTest", null, null, "com.example.FooTest" );
```

File: src/main/java/de/retest/recheck/cli/subcommands/Commit.java [hunk 2]
```diff
@@ -115,7 +114,8 @@ private void applyChanges( final Persistence<SutState> persistence, final Review
 			try {
 				ApplyChangesToStatesFlow.apply( persistence, suiteChangeSet );
 			} catch ( final NoGoldenMasterFoundException e ) {
-				logger.error( "The Golden Master '{}' cannot be found.", e.getFilename() );
+				logger.error( "The following Golden Master(s) cannot be found:\n\t- {}",
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/cli/subcommands/Commit.java]: Couldn't you just log the exception's message, as this is already formatted in `NoGoldenMasterFoundException`?

Comment (diba1013) [reply src/main/java/de/retest/recheck/cli/subcommands/Commit.java]: That would produce: 
```
Are you sure you want to continue? (y)es or (n)o:
No Golden Master file(s) with the following name(s) found:
	goldenMaster
Please make sure that the given test report '***' is within the corresponding project directory.
```

vs

```
Are you sure you want to continue? (y)es or (n)o:
The following Golden Master(s) cannot be found:
	- goldenMaster
Please make sure that the given test report '***' is within the corresponding project directory.
```

I think the current output (lower) is more understandable. Do we want to update this within recheck?

Comment (beatngu13) [reply src/main/java/de/retest/recheck/cli/subcommands/Commit.java]: I agree that the latter is better. But as you said, we probably want to have this in one place. Suggestion: Log the exception, let's fix the formatting in recheck directly later on?

**Pull request merged**